examples/platform/nxp/mcxw71_k32w1/app/support/FreeRtosHooks.c

```diff
@@ -27,6 +27,12 @@
 #include <stdio.h>
 #include <string.h>
 
+/*
+ * Temporary solution to include
```

Comment:
temporary until when? Do we have a issue for this?

Why the nogncheck?

Reply:
You need gn dependencies so gn knows that "need to compile this header before I can compile this C file". Otherwise the build is racy.

Reply:
Made the changes to include `platform/CHIPDeviceConfig.h` instead